css-text-4/Overview.bs

```diff
@@ -4603,8 +4456,41 @@ Breaking Rules for Letters: the 'word-break' property</h3>
 			(which uses [=spaces=] between words),
 			and is also useful for mixed-script text where CJK snippets are mixed
 			into another language that uses [=spaces=] for separation.
+
+		<dt><dfn>phrase</dfn>
+		<dd>
+			This value directs the user agent to perform language-specific content analysis
+			to prioritize keeping natural phrases (of multiple words) together
+			to determine [=soft wrap opportunities=]
+			and [=virtual word boundaries=].
+
+            User agents may activate
+            language-specific content analysis
+            in response to user preferences.
+            User agents with this behavior must do this
+            by setting the [=declared value=] of 'word-break' to ''word-break/phrase''
+            in the [=User Origin=].
```

Comment:
This bit should go into a separate section about accessibility tweaks that the UA can provide, not here in the core definition (where it interrupts understanding the property).